package.json

```diff
@@ -34,7 +34,7 @@
     "@carbon/icons-react": "~10.11.0",
     "@carbon/themes": "~10.12.0",
     "@data-driven-forms/pf3-component-mapper": "~2.8.13",
-    "@data-driven-forms/react-form-renderer": "~2.8.13",
+    "@data-driven-forms/react-form-renderer": "^2.8.21",
```

Comment:
Where this change is coming from?

Reply:
pretty sure that was one of the first changes made

Reply:
I don't remember, the `^` is definitely not okay and the versions of these two should always match, just make it as it was before.